packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -192,6 +196,12 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     return this._onRenderContent(tag, buttonProps);
   }
 
+  public componentDidMount() {
+    if (this._isSplitButton && this._splitButtonContainer.value && 'onpointerdown' in this._splitButtonContainer.value) {
```

Comment:
Perhaps leave a comment here on why! That way the person re-writing the button later can understand your motivation.

```
// For split buttons, touching anywhere in the button should drop the dropdown, which should contain the primary action. This gives more hit target space for touch environments.
```